bigtable/cmd/cbt/valueformatting_test.go

```diff
@@ -0,0 +1,533 @@
+/*
+Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"sort"
+	"testing"
+
+	"cloud.google.com/go/internal/testutil"
+	"github.com/jhump/protoreflect/desc"
+	"github.com/jhump/protoreflect/dynamic"
+
+	"cloud.google.com/go/bigtable"
+)
+
+func assertEqual(t *testing.T, got, want interface{}) {
+	if !testutil.Equal(got, want) {
+		_, fpath, lno, ok := runtime.Caller(1)
+		if ok {
+			_, fname := filepath.Split(fpath)
+			t.Errorf("%s:%d: Didn't match:\n%s", fname, lno, got)
+		} else {
+			t.Errorf("Didn't match:\n%s", got)
+		}
+	}
+}
+
+func assertNoError(t *testing.T, err error) bool {
```

Comment:
It's just a helper to check that there aren't errors when we don't expect them, to avoid a lot of boilerplate code.

I'm about to simplify this by using `t.Fatalf`, which I learned about yesterday. Doh! I'm fairly new to Go.

Would it help to rename this to something like `assertErrorNonNil`?